tests/utils/common.go

```diff
@@ -124,43 +129,70 @@ func IsNfs() bool {
 	return true
 }
 
-// EnableFeatureGate sets specified FeatureGate in the CDIConfig
-func EnableFeatureGate(client *cdiclientset.Clientset, feature string) (*bool, error) {
-	var previousValue = false
-	config, err := client.CdiV1beta1().CDIConfigs().Get(context.TODO(), common.ConfigName, metav1.GetOptions{})
+// UpdateCDIConfigWithOptions updates CDIConfig with specific UpdateOptions
+func UpdateCDIConfigWithOptions(c client.Client, opts metav1.UpdateOptions, updateFunc func(*cdiv1.CDIConfigSpec)) error {
+	cdi, err := controller.GetActiveCDI(c)
 	if err != nil {
-		return nil, err
+		return err
 	}
-	if HasFeature(config, feature) {
-		previousValue = true
-		return &previousValue, nil
+
+	if cdi.Spec.Config == nil {
+		cdi.Spec.Config = &cdiv1.CDIConfigSpec{}
 	}
 
-	config.Spec.FeatureGates = append(config.Spec.FeatureGates, feature)
-	config, err = client.CdiV1beta1().CDIConfigs().Update(context.TODO(), config, metav1.UpdateOptions{})
-	if err != nil {
+	updateFunc(cdi.Spec.Config)
+
+	if err = c.Update(context.TODO(), cdi, &client.UpdateOptions{Raw: &opts}); err != nil {
+		return err
+	}
+
+	if err = wait.PollImmediate(1*time.Second, 10*time.Second, func() (bool, error) {
```

Comment:
So this basically just waits to ensure that the CDIConfig has been updated for 10 seconds?

Reply:
that is the idea!